vault_program/src/cooldown_vault_ncn_ticket.rs

```diff
@@ -19,26 +19,26 @@ pub fn process_cooldown_vault_ncn_ticket(
     program_id: &Pubkey,
```

Comment:
nit: why so many changes? just need to change VaultNcnTicket::load(program_id, vault_ncn_ticket, vault, ncn, true)?;

Reply:
ocd mostly. I just renamed vault_info -> vault to match the rest of the naming conventions

Reply:
Reverted